qt/aqt/dyndeckconf.py

```diff
@@ -152,7 +152,26 @@ def on_search_button_2(self) -> None:
 
     def _on_search_button(self, line: QLineEdit) -> None:
         try:
-            search = self.mw.col.build_search_string(line.text())
+            node = self.mw.col.group_searches(
+                line.text(),
+                SearchNode(
+                    negated=SearchNode(card_state=SearchNode.CARD_STATE_SUSPENDED)
+                ),
+                SearchNode(negated=SearchNode(card_state=SearchNode.CARD_STATE_BURIED)),
+                SearchNode(negated=SearchNode(deck="filtered")),
+            )
+
+            if self.mw.col.schedVer() == 1:
+                search = self.mw.col.join_searches(
+                    node,
+                    SearchNode(
+                        negated=SearchNode(card_state=SearchNode.CARD_STATE_LEARN)
```

Comment:
This could be slightly simplified by listing the search nodes above in a list called `extra_filters`, optionally adding this one in v1, and then just calling .group_searches(line.text(), *extra_filters) once

Reply:
Actually this might need a bit more thought - skipping join_searches will wrap multiple terms in (), and if you move back and forth from the browse screen the extra filters and brackets may end up being added multiple times

Reply:
> if you move back and forth from the browse screen the extra filters and brackets may end up being added multiple times

I thought of this as I read the discussion in Rumo's PR, but guessed moving back and forth multiple times will be a rare event anyway.